### ISSUE TITLE
Custom mask function not working in nutil 4.01?

### ISSUE THREAD

**Issue by ingvildeb** [leuat/nutil#94]
In the new version of Nutil, it is no longer possible to select anything other than "QUINT" under "Analysis". Previously, there was a "none" option which disabled the atlas and xml directory so that masks could be used instead. When I try running it without anything in atlas directory but with masks instead it does not work.

**Comment by sharoncy** [comment]
@ingvildeb  we disabled the "none" option in the 4.01 release as we did not have time to implement it properly at that time (the reports, template, etc would need to be restructured etc). This is also not a priority for us now. We are considering removing this feature altogether, as it is not QUINT and is easy to perform with other image analysis software such as NIH ImageJ (I did this to validate my QUINT output in Yates et al, 2019, described in the "Validation of the Atlas Delineations" section).
